### DISCUSSION TITLE
Small tag for error messages

### DISCUSSION THREAD

**Discussion by aidamag** [dxc-technology/halstack-style-guide#710] [Q&A] [answered]
I have seen that for error messages it should be used the small tag instead of the span tag that we are using.
Should we do the change from span to small?

Example:

```
<div class="form">
     <label for="username">Username</label>
     <input type="text" id="username" placeholder="Username" />
     <small>Error message</small>
</div>        
```

**Comment by aweell** [comment, marked as answer]
Hi @Lisa159,

Can you provide examples from where you found this recommendation?

## Using <small> for error messages

It's weird because according to [<small> MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small):

> The `<small>` HTML element represents side-comments and small print, like copyright and legal text, independent of its styled presentation.

I went also through the [HTML `<small>` specs section](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element) and nothing appears related to the use case is proposed in this discussion.

I usually don't take other design system implementations as a source of truth but it seems to be no consensus in what tag should be used in these cases (e.g. I have seen implemented as a `<div>`, `<span>` or `<p>`)

## Other use cases

According to the references I previously included, I see a clear use case were where we should be using this tag thought:

* Footer's copyright element: we're currently using a `<div>` (BTW we still have 2020 as copyright year)

**Reply by aidamag** [reply]
I have learnt it from a master's subject, where it is used following an input component in a form. But reading the specs section maybe it is not very accurate to use it as an error.

About the footer, I agree that it is more suitable for the copyright (and creating right away an issue about that copyright year).
